Procfile

```diff
@@ -1,3 +1,3 @@
-release: bin/rails db:migrate
+release: bin/rails db:migrate && bin/rails search:reindex
```

Comment:
`bin/rails` accepts multiple tasks so you can do `bin/rails db:migrate search:reindex`

Reply:
True.